src/OrchardCore.Modules/OrchardCore.Notifications/Migrations/NotificationMigrations.cs

```diff
@@ -31,4 +31,18 @@ await SchemaBuilder.AlterIndexTableAsync<NotificationIndex>(table => table
 
         return 1;
     }
+
+    public async Task<int> UpdateFrom1Async()
+    {
+        await SchemaBuilder.AlterIndexTableAsync<NotificationIndex>(table => table
```

Comment:
I don't want to take part in reviewing this so in the end there will be more reviewers than lines of code :), but since I was mentioned, I'm just adding that if you add an Update method, don't forget to update the Create one as well with the same schema changes (and make it return `2`) so new installations won't have to go through both methods.